src/TodoList.js

```diff
@@ -1,5 +1,28 @@
 class TodoList {
+  constructor() {
+    this.todoList = [];
+  }
 
+  displayTodoList() {
+    return this.todoList;
+  }
+
+  create(text) {
+    const todo = {
+      id: this.todoList.length + 1,
+      text: text,
+      status: "incomplete",
+    };
+
+    this.todoList.push(todo);
+  }
+
+  isComplete(status) {
+    const statusTodo = this.todoList.filter((todo) => {
```

Comment:
`statusTodo` isn't too clear either. A good question to ask yourself here is "What does this variable contain?" - the answer: a filtered array of todos. Maybe `filteredTodos` is a better name here.

A very good solution though - this method is reusable for any status, great work